addons/mail/static/src/composer/composer.js

```diff
@@ -99,7 +99,7 @@ export class Composer extends Component {
                 );
             },
         });
-        this.suggestion = this.store.user ? useSuggestion() : undefined;
+        this.suggestion = this.store.user ? useSuggestion(this.props.composer.thread) : undefined;
```

Comment:
I removed the parameter to the hook, as the prop might changed and it refers to old thread.
The hook should read `comp.props.composer.thread` instead